src/docs/product/alerts-notifications/index.mdx

```diff
@@ -6,26 +6,40 @@ redirect_from:
 description: "Sentry categorizes notifications into alerts and other types of notifications. Learn more about how these are managed."
 ---
 
-Sentry categorizes notifications broadly into alerts and non-alert notifications.
+Alerts notify you about errors and performance problems in your application. Notifications send emails for situations like an issue state change or if a release has deployed. Alerts and notifications are customizable to your organization's needs with options to pair with [integrations](/product/integrations/), flexible configurations, and choices for both error or performance alerting. Default settings for notifications are also available.
```

Comment:
notifications cannot be sent to integrations, only alerts can.

Reply:
"choices for both alerting and performance" is repeated after you mention it in the opening sentence, would suggest remove

Reply:
"Default settings for notifications are also available." seems redundant? by definition defaults are always available